src/lib/components/Avatar/Avatar.stories.tsx

```diff
@@ -0,0 +1,49 @@
+import type { Meta, StoryObj } from "@storybook/react";
+import { Avatar } from "./Avatar";
+
+const meta = {
+  title: "Base/Avatar",
+  component: Avatar,
+  parameters: {
+    layout: "centered",
+  },
+  tags: ["autodocs"],
+  args: {
+    size: "extra large",
+    name: "Harry Roberts",
+  },
+  argTypes: {
+    size: {
+      required: true,
+      defaultValue: "extra large",
+      control: {
+        type: "radio",
+        options: ["tiny", "small", "medium", "large", "extra large"],
+      },
+    },
+    status: {
+      control: {
+        type: "radio",
+        options: [undefined, "online", "offline"],
```

Comment:
The `undefined` option doesn't work here. We need to think about different ways to unset the value.

Reply:
I looked for an answear to the problem of setting the value as undefined but i can seem to find it. I found some issues on github regarding this but from what i understand it seems that the only way is just to use the reset controlls button. I havent really used storybook so maybe there is some hack for it. Do you have any idea how to achieve the undefined option?